app/Kconfig

```diff
@@ -147,6 +147,7 @@ menuconfig ZMK_BLE
     select BT_SMP_APP_PAIRING_ACCEPT
     select BT_PERIPHERAL
     select BT_DIS
+    select BT_DEVICE_NAME_DYNAMIC
```

Comment:
I'd rather we `imply` here in case folks want to disable this.

Reply:
Addressed